VMAccess/vmaccess.py

```diff
@@ -275,6 +276,15 @@ def _set_user_account_pub_key(protect_settings, hutil):
 
     # Reset ssh key with the new public key passed in or reuse old public key.
     if cert_txt:
+        # support for SSH2-compatible format for public keys in addition to OpenSSH-compatible format
```

Comment:
You might want to add to the unit test in test_reset_account.py to cover your changes.